bootloader/ubootenv/env.go

```diff
@@ -167,6 +213,10 @@ func (env *Env) Size() int {
 	return env.size
 }
 
+func (env *Env) HeaderFlagByte() bool {
+	return env.headerFlagByte
```

Comment:
this needs to be used in tests, especially the ones testing detection when reading